File: Source/Assets/Scripts/camera/CameraTargetAttributes.js
#pragma strict
// This script goes on any GameObject in your scene that you will track with the camera.
// It'll help customize the camera tracking to your specific object to polish your game.

// See the GetGoalPosition () function in CameraScrolling.js for an explanation of these variables.
var FixedHeight				: boolean	= false;
var heightOffset 			: float 	= 0.0;
var distanceModifier 		: float 	= 2.0;
var velocityLookAhead 		: float 	= 0.15;
var maxLookAhead 			: Vector2 	= Vector2 (3.0, 3.0);

File: Source/Assets/Scripts/camera/CameraTargetAttributes.cs
using UnityEngine;
using System.Collections;


// This script goes on any GameObject in your scene that you will track with the camera.
// It'll help customize the camera tracking to your specific object to polish your game.
public class CameraTargetAttributes : MonoBehaviour {


// See the GetGoalPosition () function in CameraScrolling.js for an explanation of these variables.
public bool FixedHeight				= false;
public Vector3 Offset               = Vector2.zero;
public float distanceModifier 		= 2;
public float velocityLookAhead 		= 0.15f;
public Vector2 maxLookAhead 		= new Vector2 (3, 3);
}